From e68d797ffdfe768c1e4a356add5bb12c43f40a72 Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Fri, 9 Nov 2018 23:28:39 +0100
Subject: [PATCH] ell/ell.sym: add missing l_genl_family_set_unicast_handler

Fixes iwd compile failure:

  src/netdev.o: In function `netdev_set_nl80211':
  netdev.c:(.text+0x6f04): undefined reference to `l_genl_family_set_unicast_handler'

Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 ell/ell.sym | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ell/ell.sym b/ell/ell.sym
index 614bc93..cdd2c21 100644
--- a/ell/ell.sym
+++ b/ell/ell.sym
@@ -257,6 +257,7 @@ global:
 	l_genl_family_new;
 	l_genl_family_ref;
 	l_genl_family_unref;
+	l_genl_family_set_unicast_handler;
 	l_genl_family_set_watches;
 	l_genl_family_get_version;
 	l_genl_family_get_genl;
-- 
2.19.1

